usaspending_api/references/management/sql/restock_tas.sql

```diff
@@ -0,0 +1,239 @@
+    (
+        SELECT
+            toptier_agency_id
+        FROM
+            toptier_agency
+        WHERE
+            toptier_agency.cgac_code = broker_tas.agency_identifier
+        ORDER BY
+            toptier_agency.fpds_code
+        LIMIT 1
+    ) AS funding_toptier_agency_id,
+    broker_tas.main_account_code,
+    broker_tas.reporting_agency_aid AS reporting_agency_id,
+    broker_tas.reporting_agency_name,
+    broker_tas.sub_account_code,
+    generate_tas_rendering_label(
+        broker_tas.allocation_transfer_agency,
+        broker_tas.agency_identifier,
+        broker_tas.availability_type_code,
+        broker_tas.beginning_period_of_availa,
+        broker_tas.ending_period_of_availabil,
+        broker_tas.main_account_code,
+        broker_tas.sub_account_code
+    ) AS tas_rendering_label,
+    NOW()::DATE AS update_date
+FROM
+    dblink ('broker_server', '(
+        SELECT
+            tas_lookup.account_num::INT,
+            tas_lookup.account_title,
+            tas_lookup.agency_identifier,
+            tas_lookup.allocation_transfer_agency,
+            tas_lookup.availability_type_code,
+            tas_lookup.beginning_period_of_availa,
+            tas_lookup.budget_bureau_code,
+            tas_lookup.budget_bureau_name,
+            tas_lookup.budget_function_code,
+            tas_lookup.budget_function_title,
+            tas_lookup.budget_subfunction_code,
+            tas_lookup.budget_subfunction_title,
+            tas_lookup.ending_period_of_availabil,
+            tas_lookup.financial_indicator2,
+            tas_lookup.fr_entity_description,
+            tas_lookup.fr_entity_type,
+            tas_lookup.internal_end_date,
+            tas_lookup.internal_start_date,
+            tas_lookup.main_account_code,
+            tas_lookup.reporting_agency_aid,
+            tas_lookup.reporting_agency_name,
+            tas_lookup.sub_account_code
+        FROM
+            tas_lookup
+        WHERE
+            UPPER(tas_lookup.financial_indicator2) IS DISTINCT FROM ''F'')') AS broker_tas
+        (
+            account_num INT,
+            account_title TEXT,
+            agency_identifier TEXT,
+            allocation_transfer_agency TEXT,
+            availability_type_code TEXT,
+            beginning_period_of_availa TEXT,
+            budget_bureau_code TEXT,
+            budget_bureau_name TEXT,
+            budget_function_code TEXT,
+            budget_function_title TEXT,
+            budget_subfunction_code TEXT,
+            budget_subfunction_title TEXT,
+            ending_period_of_availabil TEXT,
+            financial_indicator2 TEXT,
+            fr_entity_description TEXT,
+            fr_entity_type TEXT,
+            internal_end_date TEXT,
+            internal_start_date TEXT,
+            main_account_code TEXT,
+            reporting_agency_aid TEXT,
+            reporting_agency_name TEXT,
+            sub_account_code TEXT
+        )
+);
+
+
+BEGIN;
```

Comment:
Unsure if this transaction block is necessary as django is calling it from a function decorated by @transaction.atomic()

Reply:
You're right, i think it's redundant. I'll remove the transaction block here.

Reply:
Can you add a big comment to the top of this SQL file that the file is not designed to be run directly but through the management command?